lib/events/onboarding.dart

```diff
@@ -177,6 +143,49 @@ class _OnboardingPageState extends State<OnboardingPage> {
     );
   }
 
+  void _isAppUpdateRequired() async {
+    setState(() => _isLoading = true);
+    try {
+      var hostPlatform = Platform.isAndroid ? "android" : "ios";
+      var forceVersion = await getApplicationConfiguration(
+          hostPlatform + '_force_update_version');
```

Comment:
Instead of doing this key creation here, I'd suggest to move it to remote config helper file and just pass the `hostPlatform` value to the new helper function.

Reply:
but that will make the function quite specific right? .. i thought of creating a function that will intake any key and will out the value against it ..